subjects/devops/input-redirection/README.md

```diff
@@ -0,0 +1,43 @@
+## Input redirection
+
+### Instructions
+
+In this exercise you will make a script `input-redirection.sh`.
+This script will read from an here document (`HereDoc`).
+Usually this technique is used to programmatically generate scripts or configuration files receiving some multiline input.
+
+The script will create a file `show_info.sh` that will run the command `cat` with `-e` as argument.
+The input to `cat` will be passed using `HereDoc`. Running `show_info.sh` will output some useful information about three common environment variables.
+
+> The environment variables are `PWD`, `PATH` and `USERNAME`.
+
+### Usage
+
+- First generate the script programmatically:
+```console
+$ ./input_redirection.sh
```

Comment:
the script name is `input-redirection.sh`

Reply:
done